rad/src/main/java/com/npr/rad/db/DaoMaster.java

```diff
@@ -112,11 +112,11 @@ public synchronized void cleanUpDb(ReportingData reportingData) {
         }
 
         if (!metadataIds.isEmpty()) {
-            metadataDao.deleteNotIn(metadataIds);
             List<String> hashes = new ArrayList<>();
             for (long id : metadataIds) {
                 hashes.add(metadataDao.getMetadata(id).getHash());
             }
+            metadataDao.deleteNotIn(metadataIds);
```

Comment:
I suggest to create a git Issue describing the topic and resolve in a separate PR.